DEVELOPMENT.md

```diff
@@ -45,6 +45,86 @@ At this point, you may find it useful to return to these `Tekton Pipeline` docs:
 -   [Tekton Pipeline "Hello World" tutorial](https://github.com/tektoncd/pipeline/blob/master/docs/tutorial.md) -
     Define `Tasks`, `Pipelines`, and `PipelineResources`, see what happens when
     they are run
+    
+## Environment Setup
+
+You must install these tools:
+
+1.  [`git`](https://help.github.com/articles/set-up-git/): For source control
+
+1.  [`go`](https://golang.org/doc/install): The language Tekton Pipelines is
+    built in
```

Comment:
```suggestion
1.  [`git`](https://help.github.com/articles/set-up-git/): For source control

2.  [`go`](https://golang.org/doc/install): The language Tekton Pipelines is
    built in
```

Reply:
Could you also specify which version of go is needed? I know that if you dont install the correct version. KO will throw a very unhelpful error.